src/services/firebase.js

```diff
@@ -22,11 +22,48 @@ import '@react-native-firebase/crashlytics';
 import '@react-native-firebase/iid';
 import '@react-native-firebase/analytics';
 import '@react-native-firebase/messaging';
-import '@react-native-firebase/remote-config';
+
+import remoteConfig from '@react-native-firebase/remote-config';
+import { INITIAL_FEATURE_FLAGS } from 'constants/featureFlagsConstants';
+import { log } from 'utils/logger';
 
 
 export const firebaseAnalytics = firebase.analytics();
 export const firebaseIid = firebase.iid();
 export const firebaseCrashlytics = firebase.crashlytics();
 export const firebaseMessaging = firebase.messaging();
 export const firebaseRemoteConfig = firebase.remoteConfig();
+
+
+export const initFirebase = async () => {
```

Comment:
Just continuing from the comment in `App.js` - We don't `await` or `return` a value here, is `async` needed?

Reply:
Reverted back - so no more waiting on both actions. Just on the first one to make sure remote config is initialised.